common/spec/spec_helper.rb

```diff
@@ -14,24 +13,6 @@
   config.raise_errors_for_deprecations!
 end
 
-VCR.configure do |config|
-  config.cassette_library_dir = "spec/fixtures/vcr_cassettes"
-  config.hook_into :webmock
-  config.configure_rspec_metadata!
-
-  # Prevent access tokens being written to VCR cassettes
-  unless ENV["DEPENDABOT_TEST_ACCESS_TOKEN"].nil?
-    config.filter_sensitive_data("<TOKEN>") do
-      ENV["DEPENDABOT_TEST_ACCESS_TOKEN"]
-    end
-  end
-
-  # Let's you set default VCR mode with VCR=all for re-recording
-  # episodes. :once is VCR default
-  record_mode = ENV["VCR"] ? ENV["VCR"].to_sym : :once
-  config.default_cassette_options = { record: record_mode }
-end
-
```

Comment:
@jspargo I've also seen the issues in tests due to VCR. VCR is still available in the main branch on dependabot/dependabot-core though, it is used by the go_module & npm_and_yarn implementations

Reply:
Yeh, that's a good point. I think I was trying this out as I thought it was interfering with webmock, but I was on the wrong track. I'll revert these changes

Reply:
I've just opened #3 to add the missing fixtures for the update_checker_spec. Tests now run completely though they still fail, probably because the pod versions in the podfile fixtures are not matching with the request fixtures